pages/desktop/developer_hub/base.py

```diff
@@ -8,14 +8,17 @@
 from selenium.webdriver.support.ui import WebDriverWait
 from selenium.webdriver.common.action_chains import ActionChains
 
+from tests.base_test import BaseTest
 from pages.page import Page
 
 
 class Base(Page):
 
-    def login(self, user="default"):
-        fxa = self.header.click_login()
-        fxa.login_user(user)
+    def login(self, mozwebqa, user="default"):
```

Comment:
I find this confusing and inconsistent. For consumer pages we changed the `login` method to just accept a user object. I think this would be cleaner and simpler if we did the same thing here.